package.json

```diff
@@ -6,6 +6,8 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "big.js": "^5.2.2",
```

Comment:
**In this project, you will be making use of `big-js` not `big.js`**
![image](https://user-images.githubusercontent.com/31897434/77390983-d472bd00-6d97-11ea-9c03-f784a94b2780.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/
**Kindly update this...**

Reply:
I installed the package using the link on the same page that you mentioned and followed the instructions in the official page. So the package should be correct.